lib/jekyll-github-metadata/owner.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module Jekyll
+  module GitHubMetadata
+    class Owner
+      extend Forwardable
+
+      # Defines an instance method that delegates to a hash's key
+      #
+      # hash   - a symbol representing the instance method to delegate to. The
+      #          instance method should return a hash or respond to #[]
+      # key    - the key to call within the hash
+      # method - (optional) the instance method the key should be aliased to.
+      #          If not specified, defaults to the hash key
+      #
+      # Returns a symbol representing the instance method
+      def self.def_hash_delegators(hash, *methods)
+        content_methods.concat(methods)
+        methods.each do |method|
+          define_method(method) do
+            send(hash)[method.to_s]
+          end
+        end
+      end
+
+      def self.content_methods
+        @content_methods ||= []
+      end
+
+      # List of whitelisted keys.
+      def_hash_delegators :owner_info,
```

Comment:
Since there is also a `private` method with the same name as `:owner_info`, which definition takes precedence?